README.md

```diff
@@ -62,6 +62,14 @@ This will yield:
 
 ```
 | uid | name  |
+|-|-|
 | 1   | Doe   |
 | 2   | Hemma |
 ```
+
+And looks like when rendered:
+
+| uid | name  |
+|-|-|
+| 1   | Doe   |
+| 2   | Hemma |
```

Comment:
This table will be rendered, so check in markdown view.

Reply:
its unnecessary to have the headers but we can add this